dev/merge_spark_pr.py

```diff
@@ -403,7 +403,7 @@ def standardize_jira_ref(text):
 
     # Extract spark component(s):
     # Look for alphanumeric chars, spaces, dashes, periods, and/or commas
-    pattern = re.compile(r'(\[[\w\s,-\.]+\])', re.IGNORECASE)
+    pattern = re.compile(r'(\[[\w\s,.-]+\])', re.IGNORECASE)
```

Comment:
Two issues: `\.` is unnecessary, and `,-.` is probably misleading as it's a range, not three characters in the class. As it happens the range from comma to period is exactly those chars in ASCII!